tests/sentry/mail/test_adapter.py

```diff
@@ -183,6 +184,75 @@ def test_simple_notification(self):
         assert msg.subject == "[Sentry] BAR-1 - Hello world"
         assert "my rule" in msg.alternatives[0][0]
 
+    def test_simple_snooze(self):
+        event = self.store_event(
+            data={"message": "Hello world", "level": "error"}, project_id=self.project.id
+        )
+
+        rule = Rule.objects.create(project=self.project, label="my rule")
```

Comment:
I recently came across the `create_project_rule` fixture if you wanna use that here